packages/block-library/src/query/edit/inspector-controls/index.js

```diff
@@ -101,6 +100,10 @@ export default function QueryInspectorControls( props ) {
 		return onChangeDebounced.cancel;
 	}, [ querySearch, onChangeDebounced ] );
 	const showInheritControl = isControlAllowed( allowedControls, 'inherit' );
+	const inheritControlLabel = __( 'Contents' );
```

Comment:
Let's go back to "Query Type" here. Although it is more technical, it's more contextual than "Contents".